README.md

```diff
@@ -1,2 +1,71 @@
+In our project we used the raycasting feature of AR Foundation to spawn the enemies perfectly on the ground. So we used a indicator instead of the default AR plane which is used to be placed on the detected plane and then based on the coordinates of the indicator we made some changes to x and z coordinates of the indicator so that the object is not exactly placed of the indicator (exactly at the place where camera is looking) but at some random point almost close to coordinates of the indicator. This procedure of spawning is being called for every 6 seconds in Level 1 and for every 5 seconds in Level 2. So, by this we were able to spawn the enemies exactly on the ground.
+We used raycasting to detect the enemies while the enemy is being killed such that enemy can be killed only when the ray hits the enemy which is in the real world and that collision coordinate is noted and then it is checked if the name of the object which was collided with the ray is with name given by us to that object and if that is matched then according to the number of shoots the enemy gets dead. So inorder to help the virtual object to be get collided with the ray we use box colliders which we attached to the body of the enemies. The green colour cube around these prefabs is the box collider. 
+
+## Limitations
+1. Real world objects like vertical planes(walls), trees and other things have not been perfectly optimized.
+2. Not compatibal for iOS yet.
+3. Bugs
+
+## Future Improvements
+1. Improve shooting effects 
+2. Adding more weapons (like throwable axe) or attacking with fist.
+3. Reduce bugs and make the game more stable.
+4. Improve the graphics and animations.
+5. Customizable graphic settings.
+6. Introduce more game objects to make the game look more scary and attractive, creating portals in the game.
+7. Improving and making the UI look more attractive.
+8. Creating more levels with higher difficulty and more different types of enemies.
+9. Creating different modes of game like time bound or limited enemies.
+10. The blood effects on camera after getting hit gets removed by shaking the device.
+11. Detecting vertical planes which should act like a hindrance for movement of enemies.
+12. Other changes in damage value of weapons and enemies to be modified.
+13. Make it compatible for iOS too.
+14. Weapons to be spawned in the world around us which have to be found to kill the enemies.
+
+![Gameplay Image!](/Images/Enemy-Death-Enemy-Chase-Blood.jpeg "Enemies")
+
+## Team Members
+- Aman Prakash Gaurav
+- Harsh Pal
+- Parameshwar Kaveti
+- Tejavath Bhavsingh
+
+## Mentors
+- Anfal Ansari
+- Rishika Chandra
+
+## References
+1. https://www.youtube.com/watch?v=Sqb-Ue7wpsI&t=17652s
+2. www.mixamo.com
+3. https://drive.google.com/drive/folders/1L2oJ8p5TppPiQD7Vw-KtCbtAPYDjI8l1?usp=sharing
+4. https://www.answers.unity.com/index.html
+5. https://www.youtube.com/watch?v=Y2ewpLX6M_s
```

Comment:
Don't include entire URL in the hyperlink. Insert a proper title & then hyperlink the URLs. Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md